### PR TITLE
[luci] Add MakeBatchNormGammaPositivePass discription

### PR DESCRIPTION
**Pull request by seanshpark**
This will add simple discription of MakeBatchNormGammaPositivePass.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
from #6054,

@jinevening , I'm not sure this discription is correct. If not, could you please add a comment?

File: compiler/luci/pass/src/MakeBatchNormGammaPositivePass.cpp [hunk 1]
```diff
@@ -69,6 +69,18 @@ bool make_positive_gamma(luci::CircleAdd *add)
 namespace luci
 {
 
+/**
+ * Make gamma value of Mul-Add(as BatchNorm) to positive
```
Comment (jinevening) [inline compiler/luci/pass/src/MakeBatchNormGammaPositivePass.cpp:73]:
```suggestion
 * Make negative gamma values of Mul-Add (as BatchNorm) to a small positive value (1e-10)
```

Comment (jinevening) [reply compiler/luci/pass/src/MakeBatchNormGammaPositivePass.cpp:73]: > @jinevening , I'm not sure this discription is correct. If not, could you please add a comment?

I slightly updated the comment :)

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Pull request merged**